src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -329,6 +329,7 @@
 
   <Node Name="BoundRefValueOperator" Base="BoundExpression">
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
+    <Field Name="NullableAnnotation" Type="NullableAnnotation" Null="disallow"/>
```

Comment:
nit: `Null="disallow"` could be removed, as it does not improve the generated code.